source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
+        history.onChangesUrl = function () {
```

Comment:
Yup, but why is it defined in `likely.js`, not in `history.js`, where all other history logics is defined? Why don’t we just write something like 

```js
// history.js
var history = {
  onUrlChanges: function () {
    if (listenChangesUrl) {
      // ...
    }
  }
}
```
?